### DISCUSSION TITLE
transition apply  for all value Iike duration only for scale not for text-colour change

### DISCUSSION THREAD

**Discussion by ArgekarStudios** [tailwindlabs/tailwindcss#6534] [Help]
I have conditional tailwind rendering. if I hover I have to add duration for scale only and text color change but the duration is applying for text color change also...

:class="[props.index===store.navIndex ?
'bg-primary shadow-lg transform scale-110 text-white' :
'text-gray-800 hover:text-white transition transform ease-out hover:bg-primary hover:shadow-lg hover:scale-110 duration-500 ']"

**Comment by reinink** [comment]
Hey there — I'm converting this to a help discussion since the issue template wasn't followed — if you want to report a bug please follow the instructions, it's just not possible for us to keep up and help if people don't do their part to give us the information we need.

**Comment by reinink** [comment]
So I'm having a hard time understanding exactly what you're trying to do, but if you're trying to apply a transition to only the scale and not to the text color, use the `transition-transform` utility (which only transitions transforms like scale), instead of `transition` (which transitions colors, transforms, filters, and more).

You can read more about this here: https://tailwindcss.com/docs/transition-property
